docs/source/user_guide/oligonucleotides_rna.rst

```diff
@@ -140,14 +152,29 @@ sequence as follows:
     for i in range(oligo_mod.size()):
         print(oligo_mod[i].isModified())
 
+Which will output
+
+.. code-block:: output
+
+
+    RNA Oligo A[m1A][Gm]A has molecular formula C42H53N20O23P3 and length 4
+    ===================================
+    RNA Oligo A[m1A][Gm]A has unmodified sequence AAGA
+    '1-methyladenosine'
+    '"'
+    'A'
+    False
+    True
+    True
+    False
 
 DNA, RNA and Protein
 ********************
 
 We can also work with DNA and RNA sequences in combination with the BioPython
 library (you can install BioPython with ``pip install biopython``):
 
-.. code-block:: pseudocode
+.. code-block:: python
```

Comment:
You probably have to be careful. All python code-blocks will be tested. Not sure if this code is functional.
Just saying, you might need to fix it in CI.

Reply:
I reverted to pseudocode, just to avoid installing biopython, which is rather large

Reply:
yep. fine with me